test/build2file

```diff
@@ -39,6 +37,9 @@ for test_cpp : $tests_sources
 if($cxx.id != 'msvc')
 {
     # Enable all warnings:
+    # cja my clang 15.0 pre-release on Windows doesn't like -pedantic-errors...
```

Comment:
what do you mean by "doesnt like"?

Reply:
As far as I can tell, it behaves as if it's not a valid command line argument (despite the fact that it's documented).
`b -v` prints the `clang++` command line and then reports `error: unable to preprocess ...`. With this argument removed, all is fine.

Reply:
That's weird indeed.